cypress/support/tasks/logout.js

```diff
@@ -10,4 +11,5 @@ export function navigateUserMenu(){
 export function logoutButton(){
   cy.get(Logout_page.logout_button).click();
   cy.url().should('include','ipd/welcome');
+  cy.get(Login_page.button_welcome).should('be.visible').should('be.enabled') 
```

Comment:
can we at least alight on tab and space for each function please, later on I will try to put eslint in.

Reply:
Now i use prettier to format all file in project.